### PR TITLE
par: update to 1.53.0

### PR DESCRIPTION
**Pull request by casr**
#### Description

  - use makefile port group (for universal support)
  - remove patches that have been folded into this release
  - add test phase although disable some tests that fail
  - add livecheck

See: http://www.nicemice.net/par/releasenotes.var

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.0.1 21A559 arm64
Xcode 13.1 13A1030d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: textproc/par/Portfile [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 
-# Based on: http://www.openbsd.org/cgi-bin/cvsweb/ports/textproc/par/Makefile
-
 PortSystem          1.0
+PortGroup           makefile 1.0
 
 name                par
-version             1.52
-revision            2
+version             1.53.0
```
Comment (pmetzger) [inline textproc/par/Portfile:7]: Revision should be set to 0 rather than removed...

**Review by pmetzger — CHANGES_REQUESTED** [review]
Howdy! Unclear to me if the maintainer is still active given that only their email is in the Portfile. I don't know enough to review most of this (hopefully others will chime in) but I do have one very minor comment to make.

**Comment by casr** [comment]
I remember when I last made changes to this port that Quentin Barnes chimed in. Their email matches a GitHub profile at @qbarnes so this should hopefully serve as enough of a ping.

Also, to give some context, this project was dormant for 19 years and then randomly received an update in 2020.

File: textproc/par/Portfile (hunk 1 above)
Comment (casr) [reply textproc/par/Portfile:7]: Added back.

**Comment by herbygillot** [comment]
> Their email matches a GitHub profile at @qbarnes so this should hopefully serve as enough of a ping.

If that's confirmed, the Github handle should be added to the `maintainers` block.

**Comment by qbarnes** [comment]
I am the same person as the `par` maintainer for macports.

This is not a good commit. It removes the i18n functionality from `par` that some users critically rely on.  Please see all the comments under https://trac.macports.org/ticket/63007.  At least back 7 months ago the author of `par` (Adam M. Costello) knew of no one porting i18n functionality to 1.53.0.  I tried for a bit, but gave up returning to other projects.  Any volunteers?

If there are any volunteers, I'll bring them up to speed on the progress and problems I ran into.

**Comment by casr** [comment]
Thanks for linking the ticket on Trac. I tried to see if there is anything there but the search can be limiting sometimes...

I must admit that my main motivation for taking a look at this port was to add `+universal` support to it and I thought the upgrade would be a nice bonus. I will close this and perhaps just do the makefile port group addition in a follow up PR.

@qbarnes Would you like me to add in your GitHub handle to the maintainers in that follow up PR?

**Comment by qbarnes** [comment]
Decoupling the version change from the other cleaning work on the `Portfile` is a good idea.

Yes, please add me.

**Comment by casr** [comment]
https://github.com/macports/macports-ports/pull/13534

**Comment by pmetzger** [comment]
@qbarnes Can we get your github handle added to the portfile?

**Comment by qbarnes** [comment]
I'm kinda confused.  What would my github handle be if its not the @qbarnes you tagged me with?

**Comment by casr** [comment]
> I'm kinda confused. What would my github handle be if its not the @qbarnes you tagged me with?

I think @pmetzger likely missed your previous acceptance.

> Yes, please add me.

**Comment by pmetzger** [comment]
@qbarnes Your github handle isn't in the portfile.

**Comment by pmetzger** [comment]
Portfile only says:

```
maintainers         gmail.com:qbarnes
```

and should probably say

```
maintainers         {@qbarnes gmail.com:qbarnes} openmaintainer
```

**Pull request closed without merging**